gremlinapi/scenario_helpers.py

```diff
@@ -24,50 +24,54 @@
 
```

Comment:
TODO: Evaluate deprecating this interface. IIRC the API no longs take the output this helper generates.

Reply:
Will depreciate entire scenario_helpers.py file